broker/utils.go

```diff
@@ -25,10 +25,6 @@ func findDeployment(c compose.Client, name string) (*composeapi.Deployment, erro
 	return nil, fmt.Errorf("deployment: not found")
 }
 
-func JDBCURI(scheme, hostname, port, dbname, username, password string) string {
```

Comment:
yes, that's true if the future is unknown and distant. We've got stories scheduled for the sprint starting 19th of July to add ES support :). Do we want to revert this commit there?